src/systemd/cockpit-wsinstance-http.socket

```diff
@@ -3,11 +3,12 @@ Description=Socket for Cockpit Web Service http instance
 Documentation=man:cockpit-ws(8)
 BindsTo=cockpit.service
 # ensure our DynamicUser exists
-Requires=cockpit-ws-user.service
-After=cockpit-ws-user.service
+Requires=cockpit-ws-user.service cockpit-wsinstance-socket-user.service
+After=cockpit-ws-user.service cockpit-wsinstance-socket-user.service
```

Comment:
Likewise, this shouldn't be here. service socket depending on its own user (i.e. status quo) is ok of course, but not for foreign services.